requirements.in

```diff
@@ -20,7 +20,7 @@ factory-boy
 fakeredis
 freezegun
 https://github.com/codecov/opentelem-python/archive/refs/tags/v0.0.4a1.tar.gz#egg=codecovopentelem
-https://github.com/codecov/shared/archive/1dc95a2ac3c07ca211319e65a90ea35592c825c1.tar.gz#egg=shared
+https://github.com/codecov/shared/archive/9ca2efbb2c109ecfe241143ecd1c6f38c3820a0b.tar.gz#egg=shared
```

Comment:
Don't merge until updated to latest after this is merged, https://github.com/codecov/shared/pull/214